qt/applications/workbench/workbench/plugins/editor.py

```diff
@@ -9,16 +9,16 @@
 #
 from __future__ import (absolute_import, unicode_literals)
 
-# system imports
+from qtpy.QtCore import Qt
```

Comment:
Any reason moving this? It'd be nice to keep the order: stdlib modules, 3rdparty modules, local modules.